### ISSUE TITLE
🔥🔥🔥some questions?

### ISSUE THREAD

**Issue by Peterisfar** [JiaRenChang/PSMNet#181]
1、why sceneflow dataset need to pad in the test, but kitti2015 don't need?

2、code about computing 3-px error 
` true_disp = disp_true`
their memory address are same, so 
`correct = (disp_true[index[0][:], index[1][:], index[2][:]] < 3) | (
                disp_true[index[0][:], index[1][:], index[2][:]] < true_disp[
            index[0][:], index[1][:], index[2][:]] * 0.05)` is wrong.

**Comment by Peterisfar** [comment]
3、in the finetune.py, 
`        if total_test_loss / len(TestImgLoader) * 100 > max_acc:
            max_acc = total_test_loss / len(TestImgLoader) * 100
            max_epo = epoch`

`# computing 3-px error#
    true_disp = disp_true
    index = np.argwhere(true_disp > 0)
    disp_true[index[0][:], index[1][:], index[2][:]] = np.abs(
        true_disp[index[0][:], index[1][:], index[2][:]] - pred_disp[index[0][:], index[1][:], index[2][:]])
    correct = (disp_true[index[0][:], index[1][:], index[2][:]] < 3) | (
                disp_true[index[0][:], index[1][:], index[2][:]] < true_disp[
            index[0][:], index[1][:], index[2][:]] * 0.05)
    torch.cuda.empty_cache()

    return 1 - (float(torch.sum(correct)) / float(len(index[0])))`


you test return the error between the pred and GT，but when you in the main() you see the max error as max acc, why? is it the error bigger the model' weight is worse. May you explain the reason?

**Comment by Peterisfar** [comment]
4、why in the main, you set the mask below the args.maxdisp; but in the finetune, you set the mask up the 0? why?

**Comment by Peterisfar** [comment]
NOTE:
about question 1  and 4,  i have known the reason.

**Comment by kaynezhangqy** [comment]
> 3、in the finetune.py, ` if total_test_loss / len(TestImgLoader) * 100 > max_acc: max_acc = total_test_loss / len(TestImgLoader) * 100 max_epo = epoch`
> 
> ![image](https://user-images.githubusercontent.com/31852119/80908465-a5664880-8d52-11ea-9455-b5643956aaf5.png)
> 
> you test return the error between the pred and GT，but when you in the main() you see the max error as max acc, why? is it the error bigger the model' weight is worse. May you explain the reason?

I aslo notice this, and I am confused about it ! @JiaRenChang
